### PR TITLE
Modify codes about version info

### PR DESCRIPTION
**Pull request by dreamerlin**
This PR modifies codes related with some version information.
1. Remove git hash in  `__version__`.
2. Check `mmcv` version.

### CODE REVIEW

File: mmaction/__init__.py [hunk 1]
```diff
@@ -1,3 +1,28 @@
+import mmcv
+
 from .version import __version__, short_version
 
+
+def digit_version(version_str):
```
Comment (innerlee) [inline mmaction/__init__.py:6]: can this version thing be moved to mmcv?

Comment (dreamerlin) [reply mmaction/__init__.py:6]: Agree, since it is also used in `mmdet`.

File: mmaction/__init__.py [hunk 2]
```diff
@@ -1,3 +1,15 @@
+import mmcv
+from mmcv import digit_version
+
 from .version import __version__, short_version
 
+mmcv_minimum_version = '1.1.1'
+mmcv_maximum_version = '1.2'
```
Comment (innerlee) [inline mmaction/__init__.py:7]: upper bound a dependency is not a good idea

File: mmaction/__init__.py (hunk 1 above)
Comment (dreamerlin) [reply mmaction/__init__.py]: Found the `digit_version` has already existed in `mmcv`, so I directly import it here.

File: mmaction/__init__.py (hunk 2 above)
Comment (dreamerlin) [reply mmaction/__init__.py:7]: Agree.

File: mmaction/utils/collect_env.py [hunk 3]
```diff
@@ -1,11 +1,12 @@
 from mmcv.utils import collect_env as collect_basic_env
+from mmcv.utils import get_git_hash
 
 import mmaction
 
 
 def collect_env():
     env_info = collect_basic_env()
-    env_info['MMAction2'] = mmaction.__version__
+    env_info['MMAction2'] = mmaction.__version__ + '+' + get_git_hash()[:7]
```
Comment (innerlee) [inline mmaction/utils/collect_env.py:9]: can you move this `[:7]` into the `get_git_hash` function?

Comment (innerlee) [reply mmaction/utils/collect_env.py:9]: or give a new `get_short_git_hash` function

File: tools/train.py [hunk 4]
```diff
@@ -135,7 +136,7 @@ def main():
         # save mmaction version, config file content and class names in
         # checkpoints as meta data
         cfg.checkpoint_config.meta = dict(
-            mmaction_version=__version__, config=cfg.text)
+            mmaction_version=__version__ + get_git_hash()[:7], config=cfg.text)
```
Comment (innerlee) [inline tools/train.py:139]: here

**Comment by innerlee** [comment]
CU-4dnmw1

**Comment by hellock** [comment]
Task linked: [CU-4dnmw1 MMAction2](https://app.clickup.com/t/4dnmw1)

**Comment by hellock** [comment]
Task linked: [CU-4dnmxx MMAction2](https://app.clickup.com/t/4dnmxx)

File: mmaction/utils/collect_env.py [hunk 5]
```diff
@@ -1,11 +1,19 @@
 from mmcv.utils import collect_env as collect_basic_env
+from mmcv.utils import get_git_hash
 
 import mmaction
 
+NUM_HASHES = 7
+
+
+def get_short_git_hash(num_hashes=7):
```
Comment (innerlee) [inline mmaction/utils/collect_env.py:9]: can this function be moved to mmcv?

Comment (innerlee) [reply mmaction/utils/collect_env.py:9]: let's do it in a separate pr

**Review by innerlee — APPROVED** [review]

File: mmaction/version.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Open-MMLab. All rights reserved.
+
+__version__ = '0.6.0'
+short_version = __version__
+
+
+def parse_version_info(version_str):
```
Comment (innerlee) [inline mmaction/version.py:7]: this should also go to mmcv

Comment (innerlee) [reply mmaction/version.py:7]: cc. @hellock

File: mmaction/utils/collect_env.py (hunk 5 above)
Comment (innerlee) [reply mmaction/utils/collect_env.py:9]: cc. @hellock

**Review by innerlee — APPROVED** [review]

File: mmaction/utils/collect_env.py [hunk 7]
```diff
@@ -1,11 +1,19 @@
 from mmcv.utils import collect_env as collect_basic_env
+from mmcv.utils import get_git_hash
 
 import mmaction
 
+NUM_HASHES = 7
```
Comment (innerlee) [inline mmaction/utils/collect_env.py:6]: this is not needed any more

File: mmaction/utils/collect_env.py [hunk 8]
```diff
@@ -1,11 +1,19 @@
...
+
+def get_short_git_hash(num_hashes=7):
+    return get_git_hash()[:num_hashes]
+
 
 def collect_env():
     env_info = collect_basic_env()
-    env_info['MMAction2'] = mmaction.__version__
+    env_info['MMAction2'] = (
+        mmaction.__version__ + '+' + get_short_git_hash(NUM_HASHES))
```
Comment (innerlee) [inline mmaction/utils/collect_env.py:16]: same here

**Pull request merged**